install/conf/srm-toolchain/prometheus/alerts.rules

```diff
@@ -0,0 +1,62 @@
+groups:
+- name: Throughput
+  rules:
+
+  - alert: high_throughput
```

Comment:
Some of these are very specific to Delfin right?! Why these confs should be added separately under the dashboard?! Can we not source it from the respective project?!

Reply:
These configurations are for configuring Prometheus and alertmanager to be viewed using SODA Dashboard. User may not use this if they are not using SODA Dashboard. Hence the configurations are to be kept in the Dashboard or SRM toolchain conf.

Reply:
I think it should be the installer configuration either from SODA installer or Delfin installer where the user decides to have dashboard or not.

Reply:
We can consider this as an enhancement. Currently we are creating independent installer for Dashboard, which can be run from the Dashboard release.
Since most of these are environment variables the SODA Installer can just configure them at the time of installation. (This is how it currently works in the common installer as of v1.7.0)